eos/b-decays/b-to-psd-l-nu_TEST.cc

```diff
@@ -0,0 +1,134 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2018, 2019 Ahmet Kokulu
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/observable.hh>
+#include <eos/b-decays/b-to-d-l-nu.hh>
+#include <eos/utils/complex.hh>
+#include <eos/utils/wilson-polynomial.hh>
+
+#include <array>
+#include <cmath>
+#include <iostream>
+#include <fstream>
+#include <limits>
+#include <string>
+#include <vector>
+
+
+using namespace test;
+using namespace eos;
+
+class BToDLeptonNeutrinoTest :
+    public TestCase
+{
+    public:
+        BToDLeptonNeutrinoTest() :
+            TestCase("b_to_d_l_nu_test")
+        {
+        }
+
+        virtual void run() const
+        {
+            // SM tests
+            {
+                Parameters p1 = Parameters::Defaults();
+                p1["B->D::f_+(0)@BCL2008"]  = +0.660;
+                p1["B->D::f_T(0)@BCL2008"]  = +0.00;
+                p1["B->D::b_+^1@BCL2008"]   = -4.00;
+                p1["B->D::b_+^2@BCL2008"]   = -0.80;
+                p1["B->D::b_0^1@BCL2008"]   = +0.40;
+                p1["B->D::b_0^2@BCL2008"]   = -1.20;
+                p1["B->D::b_T^1@BCL2008"]   = +0.00;
+                p1["B->D::b_T^2@BCL2008"]   = +0.00;
+                p1["mass::B_d"]             =  5.279;
+                p1["mass::D_d"]             =  1.870;
+                // by default, all other couplings are zero in eos
+                p1["b->cmunumu::Re{cVL}"]   =  1.0;
+
+                Options oo;
+                oo.set("model", "WilsonScan");
+                oo.set("form-factors", "BCL2008");
+                oo.set("q", "d");
```

Comment:
Set also options "U" and "l" to not to rely on current default settings?

Reply:
We decided anyway to remove this test case, since it duplicates the existing B->Dlnu test case...